spring-cloud-dataflow-docs/src/main/asciidoc/shell.adoc

```diff
@@ -86,77 +84,77 @@ Description:               Create a new stream definition
 
-  transform --expression='new StringBuilder(payload).reverse()'
+`transform --expression='new StringBuilder(payload).reverse()'`
 
-If the parameter value needs to embed a single quote, use two single quotes:
+If the parameter value needs to embed a single quote, use two single quotes, as follows:
 
-  // Query is: Select * from /Customers where name='Smith'
-  scan --query='Select * from /Customers where name=''Smith'''
+[source]
+// Query is: Select * from /Customers where name='Smith'
+scan --query='Select * from /Customers where name=''Smith'''
 
 
 [[dsl-quotes-escaping]]
 === Quotes and Escaping
 
-There is a *Spring Shell based client* that talks to the Data Flow Server that is responsible for *parsing* the DSL.
+There is a Spring Shell-based client that talks to the Data Flow Server and is responsible for *parsing* the DSL.
 In turn, applications may have applications properties that rely on embedded languages, such as the *Spring Expression Language*.
 
 The shell, Data Flow DSL parser, and SpEL have rules about how they handle quotes and how syntax escaping works.
 When combined together, confusion may arise.
-This section explains the rules that apply and provides examples of the most complicated situations you will encounter when all three components are involved.
+This section explains the rules that apply and provides examples of the most complicated situations you may encounter when all three components are involved.
 
 [NOTE]
 .It's not always that complicated
 ====
-If you don't use the Data Flow shell, for example you're using the REST API directly, or if applications properties are not SpEL expressions, then escaping rules are simpler.
+If you do not use the Data Flow shell (for example, you use the REST API directly) or if applications properties are not SpEL expressions, then the escaping rules are simpler.
 ====
 
-
 ==== Shell rules
 Arguably, the most complex component when it comes to quotes is the shell. The rules can be laid out quite simply, though:
 
-* a shell command is made of keys (`--foo`) and corresponding values. There is a special, key-less mapping though, see below
-* a value can not normally contain spaces, as space is the default delimiter for commands
-* spaces can be added though, by surrounding the value with quotes (either single [`'`] or double [`"`] quotes)
-* if surrounded with quotes, a value can embed a literal quote of the same kind by prefixing it with a backslash (`\`)
-* Other escapes are available, such as `\t`, `\n`, `\r`, `\f` and unicode escapes of the form `\uxxxx`
-* Lastly, the key-less mapping is handled in a special way in the sense that if does not need quoting to contain spaces
+* A shell command is made of keys (`--something`) and corresponding values. There is a special, keyless mapping, though, which is described later.
+* A value can not normally contain spaces, as space is the default delimiter for commands.
```

Comment:
"can not" -> "cannot"?

Reply:
https://www.grammarly.com/blog/cannot-or-can-not/